pkgs/servers/misc/podgrab/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "podgrab";
+  version = "2021-04-14"; # No official version tags
+
+  src = fetchFromGitHub {
+    owner = "akhilrex";
+    repo = pname;
+    rev = "3179a875b8b638fb86d0e829d12a9761c1cd7f90";
+    sha256 = "sha256-vhxIm20ZUi+RusrAsSY54tv/D570/oMO5qLz9dNqgqo=";
+  };
+
+  vendorSha256 = "sha256-xY9xNuJhkWPgtqA/FBVIp7GuWOv+3nrz6l3vaZVLlIE=";
+
+  postInstall = ''
+    mkdir -p $out/share/
+    ln -s -r "$src/client" "$out/share/"
```

Comment:
You probably want to actually copy these to $out rather than symlinking them; that means that users of the output won't end up needlessly pulling down the sources as well.